Source/Core/DolphinQt2/Config/GeckoCodeWidget.h

```diff
@@ -46,6 +46,7 @@ class GeckoCodeWidget : public QWidget
   void RemoveCode();
   void DownloadCodes();
   void SaveCodes();
+  void SortCode();
```

Comment:
`SortCodesLexicographically()` would also indicate the sorting type being performed (i.e. `operator<` on two strings is defined as a general lexicographical comparison).